integration_tests/integration/documents.spec.ts

```diff
@@ -0,0 +1,31 @@
+import Page from '../pages/page'
+import DocumentsPage from '../pages/documents'
+
+context('Documents', () => {
```

Comment:
Could do with tests to check the instructions text box keeps the text - but if the other pages don't have it then maybe not worth it?! Maybe have a separate test to cover the instructions text box requirements if we don't already have one.